config/settings.yml

```diff
@@ -5,3 +5,5 @@ validate:
     maximum: 255
   password:
     minimum: 6
+gravatar:
+  size: 80
```

Comment:
Please add a line break at the end of this file. You can understand this if you see other files ending